vcf/fix.go

```diff
@@ -19,18 +19,22 @@ func FixVcf(query *Vcf, ref map[string][]dna.Base) {
 // to represent a deletion. This function will reformat the vcf record
 // according to VCF specs
 func fixDash(query *Vcf, ref map[string][]dna.Base) {
-	if strings.Compare(query.Alt, "-") == 0 {
-		// query.Pos is -2, one for zero base, and one for previous base
-		prevBase := dna.BaseToString(ref[query.Chr][query.Pos-2])
-		query.Pos--
-		query.Ref = prevBase + query.Ref
-		query.Alt = prevBase
+	for i := 0; i < len(query.Alt); i++ {//TODO: (Riley) right now we're just checking if any of the alts have dashes. Not sure if this is the best way to handle polyallelic sites for fix. 
+			if strings.Compare(query.Alt[i], "-") == 0 {
+			// query.Pos is -2, one for zero base, and one for previous base
+			prevBase := dna.BaseToString(ref[query.Chr][query.Pos-2])
```

Comment:
can query.Pos-2 be negative?

Reply:
I only changed castings here, this logic is not something I wrote.